pvlib/iotools/pvgis.py

```diff
@@ -665,3 +666,42 @@ def read_pvgis_tmy(filename, pvgis_format=None, map_variables=None):
         data = data.rename(columns=VARIABLE_MAP)
 
     return data, months_selected, inputs, meta
+
+
+def get_pvgis_horizon(latitude, longitude, proxies=None, url=URL):
+    r'''
+    Get horizon data from PVGIS
+    Parameters
+    ----------
+    latitude : float
+        Latitude in degrees north
+    longitude : float
+        Longitude in degrees east
+    proxies: dict
+        Dictionary of proxies to access through a corporate network
+    url: string
+        Base URL for PVGIS
+
+    Returns
+    -------
+    df : pd.DataFrame
+        Pandas dataframe of the retrived horizon
+    '''
+    res = requests.get(url +
+                       f'printhorizon?lat={latitude}&lon={longitude}',
+                       proxies=proxies, verify=False)
```

Comment:
Same comment regarding letting user splat in arbitrary `**kwargs` to deal with whatever headaches their connection might be subject to